translate/cloud-client/src/main/java/com/example/translate/HybridGlossaries.java

```diff
@@ -0,0 +1,301 @@
+import com.google.cloud.texttospeech.v1.SynthesisInput;
+import com.google.cloud.texttospeech.v1.SynthesizeSpeechResponse;
+import com.google.cloud.texttospeech.v1.TextToSpeechClient;
+import com.google.cloud.texttospeech.v1.VoiceSelectionParams;
+
+import com.google.cloud.translate.v3beta1.CreateGlossaryRequest;
+import com.google.cloud.translate.v3beta1.GcsSource;
+import com.google.cloud.translate.v3beta1.Glossary;
+import com.google.cloud.translate.v3beta1.Glossary.LanguageCodesSet;
+import com.google.cloud.translate.v3beta1.GlossaryInputConfig;
+import com.google.cloud.translate.v3beta1.GlossaryName;
+import com.google.cloud.translate.v3beta1.LocationName;
+import com.google.cloud.translate.v3beta1.TranslateTextGlossaryConfig;
+import com.google.cloud.translate.v3beta1.TranslateTextRequest;
+import com.google.cloud.translate.v3beta1.TranslateTextResponse;
+import com.google.cloud.translate.v3beta1.TranslationServiceClient;
+
+import com.google.cloud.vision.v1.AnnotateImageRequest;
+import com.google.cloud.vision.v1.AnnotateImageResponse;
+import com.google.cloud.vision.v1.BatchAnnotateImagesResponse;
+import com.google.cloud.vision.v1.Feature;
+import com.google.cloud.vision.v1.Feature.Type;
+import com.google.cloud.vision.v1.Image;
+import com.google.cloud.vision.v1.ImageAnnotatorClient;
+
+import com.google.common.html.HtmlEscapers;
+import com.google.protobuf.ByteString;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+// [END translate_hybrid_imports]
+
+
+public class HybridGlossaries {
+
+  // [START translate_hybrid_project_id]
+  private static final String projectId = System.getenv("PROJECT_ID");
+  // [END translate_hybrid_project_id]
+
+  // [START translate_hybrid_vision]
+  /**
+   * @param filePath input image file
+   **/
+  static String picToText(String filePath) {
+    List<AnnotateImageRequest> requests = new ArrayList<>();
+
+    try (ImageAnnotatorClient client = ImageAnnotatorClient.create()) {
+      ByteString imageBytes = ByteString.readFrom(new FileInputStream(filePath));
+
+      Image image = Image.newBuilder().setContent(imageBytes).build();
+
+      Feature feature = Feature.newBuilder().setType(Type.DOCUMENT_TEXT_DETECTION).build();
+
+      AnnotateImageRequest request =
+              AnnotateImageRequest.newBuilder()
+                      .addFeatures(feature)
+                      .setImage(image)
+                      .build();
+
+      requests.add(request);
+
+      BatchAnnotateImagesResponse response = client.batchAnnotateImages(requests);
+      List<AnnotateImageResponse> responses = response.getResponsesList();
+
+      StringBuilder result = new StringBuilder();
+
+      for (AnnotateImageResponse res : responses) {
+        result.append(String.format("%s\n", res.getFullTextAnnotation().getText()));
+      }
+
+      return result.toString();
```

Comment:
Preferable we don't return, and just process the content somehow (log). See [Function Return Type](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#function-return-type).

Reply:
This is still returning

Reply:
Yea as part of the tutorial this gets sent over to the next method.

Reply:
Is this a hard required change for the tutorial?
I'll need to work with the TW to update the docs and other languages if so.

Reply:
I think this is fine since we are already bending a few other rules here. At this point it's more like  a sample application.